doc/api_assets/style.css

```diff
@@ -330,6 +330,27 @@ hr {
   margin-top: .666em;
 }
 
+#toc .stability_0::after,
+#toc .stability_0 a:hover,
+#toc .stability_0 a:focus {
+  background-color: #d50027;
+  color: #fff;
+}
+
+#toc .stability_0::after {
```

Comment:
> This may be better as ::before

This cannot help with current realization :(
Pseudo-element attaches for list item's child (tag `<a>`) and cannot influence to somethings. I will try to find other solution in days